packages/borgbackup/use-distutils.patch

```diff
@@ -1,83 +0,0 @@
-From 2ae2e3a15fa3e361b5f2e186d2450a8ccd38f698 Mon Sep 17 00:00:00 2001
-From: Leonid Pliushch <leonid.pliushch@gmail.com>
-Date: Mon, 26 Jul 2021 13:30:08 +0300
-Subject: [PATCH] Termux borgbackup: use distutils instead of packaging
-
-Do not remove this patch unless you know what you are doing!
-
-We don't have module "packaging" packaged and thus cannot use it without
-installing through "pip". But for stable packages we cannot use it in
-postinst script to add missing dependencies.
-
-So if you decide to drop this patch:
-* Add python3-packaging to termux-packages.
-* Alternatively: create a postinst deb script (pip3 install packaging)
-  and move borgbackup to https://github.com/termux/unstable-packages
----
- setup.py             | 24 ++++++++----------------
- src/borg/__init__.py |  5 ++---
- src/borg/xattr.py    |  6 +++---
- 3 files changed, 13 insertions(+), 22 deletions(-)
-
-diff --git a/pyproject.toml b/pyproject.toml
-index 9ecaccb6..d81de77f 100644
---- a/pyproject.toml
-+++ b/pyproject.toml
-@@ -35,7 +35,6 @@ dependencies = [
-     # using any other msgpack version is not supported by borg development and
-     # any feedback related to issues caused by this will be ignored.
-     "msgpack >=1.0.3, <=1.1.0",
--    "packaging",
- ]
- 
- # note for package maintainers: if you package borgbackup for distribution,
-diff --git a/setup.py b/setup.py
-index 1096d1ad..bcd2acf5 100644
---- a/setup.py
-+++ b/setup.py
-@@ -10,9 +10,11 @@
- except ImportError:
-     multiprocessing = None
- 
--from setuptools.command.build_ext import build_ext
--from setuptools import setup, Extension
-+from distutils.command.build_ext import build_ext
-+from setuptools import setup, find_namespace_packages, Extension
- from setuptools.command.sdist import sdist
-+from distutils.core import Command
-+from distutils.command.clean import clean
- 
- try:
-     from Cython.Build import cythonize
-diff --git a/src/borg/__init__.py b/src/borg/__init__.py
-index 9f13c7d6..47adb1b6 100644
---- a/src/borg/__init__.py
-+++ b/src/borg/__init__.py
-@@ -1,12 +1,11 @@
--from packaging.version import parse as parse_version
-+from distutils.version import LooseVersion
- 
- # IMPORTANT keep imports from borg here to a minimum because our testsuite depends on
- # being able to import borg.constants and then monkey patching borg.constants.PBKDF2_ITERATIONS
- from ._version import version as __version__
- 
- 
--_v = parse_version(__version__)
--__version_tuple__ = _v._version.release
-+__version_tuple__ = tuple(LooseVersion(__version__).version[:3])
- 
- # assert that all semver components are integers
- # this is mainly to show errors when people repackage poorly
-diff --git a/src/borg/xattr.py b/src/borg/xattr.py
-index 76e3b8d4..5d3bf14c 100644
---- a/src/borg/xattr.py
-+++ b/src/borg/xattr.py
-@@ -7,7 +7,7 @@
- import sys
- import tempfile
- 
--from packaging.version import parse as parse_version
-+from distutils.version import LooseVersion as parse_version
```

Comment:
Can we actually get rid of this part of the patch just by switching to PEP-518/pyproject.toml?